### PR TITLE
Complete documentation, refactor gesture, update for segment picker, …

### PR DESCRIPTION
**Pull request by kevinwl02**
…jazzy.yaml

### CODE REVIEW

File: AltSwiftUI/AltSwiftUI/Source/Views/Collections/ViewCollectionIteration.swift [hunk 1]
```diff
@@ -138,10 +138,12 @@ extension Array where Element == View {
             // Optional insert / update
             let oldOptionalView = oldView as? OptionalView
             let maxCount = Swift.max(optionalViewContent.count, oldOptionalView?.content?.count ?? 0)
-            for i in 0..<maxCount {
-                let subView = optionalViewContent[safe: i]
-                let oldSubView = oldOptionalView?.content?[safe: i]
-                iterateFullSubviewDiff(subView: subView, oldView: oldSubView, iteration: iteration, displayIndex: &displayIndex)
+            if maxCount > 0 {
```
Comment (AlexTanabe) [inline AltSwiftUI/AltSwiftUI/Source/Views/Collections/ViewCollectionIteration.swift]: Do you need this if? I mean, `for i in 0..<0 {` will not execute anyway right? 🤔

**Review by AlexTanabe — APPROVED** [review]
Added one comment but is still approved overall.

File: AltSwiftUI/AltSwiftUI/Source/Views/Collections/ViewCollectionIteration.swift (hunk 1 above)
Comment (kevinwl02) [reply AltSwiftUI/AltSwiftUI/Source/Views/Collections/ViewCollectionIteration.swift]: you are right, `0..<0` won't crash. Fixed.

**Review by AlexTanabe — COMMENTED** [review]
LGTM~

**Pull request merged**